src/main/java/com/Incubatormsrv/Incubatormsrv/weather/WeatherApiException.java

```diff
@@ -0,0 +1,7 @@
+package com.Incubatormsrv.Incubatormsrv.weather;
+
+public class WeatherApiException extends RuntimeException {
```

Comment:
Please annotate with the status that you want this exception to give back... e.g. 500